src/dvclive/dvc.py

```diff
@@ -108,3 +123,33 @@ def make_dvcyaml(live):
     if live._metrics or live._plots or live._images:
         dvcyaml["plots"] = [os.path.relpath(live.plots_dir, live.dir)]
     dump_yaml(dvcyaml, live.dvc_file)
+
+
+def mark_dvclive_only_started():
```

Comment:
[Q] Are there better names for these functions `mark_dvclive_only_started` & `mark_dvclive_only_ended`?

Reply:
Probably 😅 but It is ok for me to merge as it is